kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/wlst/create-domain.py

```diff
@@ -102,6 +102,12 @@ def getEnvVar(var):
   cmo.setCluster(cl)
   print('Done setting attributes for Server Template: %s' % templateName);
 
+  templateChannelName = cluster_name + "-NAP"
```

Comment:
You only need to change integration-tests/src/test/resources/domain-home-on-pv/create-domain.py. I think the change to kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/wlst/create-domain.py is not needed.

Reply:
@vanajamukkara Thank you! My bad, that was a remnant from when the tests was using the wrong domain creation scripts. I took it out.